syntax = "proto3";

enum CrossShardResultType {
  YES = 0; // Prepare phase
  NO = 1; // Prepare phase
  ACK = 2; // Commit/Abort phase
}

message CrossShardRsp {
  CrossShardResultType result = 1;
  int32 id = 2;
}